fluentd/configs.d/filter/k8s_record_transform.conf

```diff
@@ -1,4 +1,24 @@
-<filter kubernetes.**>
+<filter kubernetes.journal.container**>
+  type record_transformer
+  enable_ruby
+  <record>
+    hostname ${(kubernetes_host rescue nil) || File.open('/etc/docker-hostname') { |f| f.readline }.rstrip}
+    # log is set if MESSAGE is JSON valued and has err or msg field
+    message ${record['MESSAGE'] || log}
+    version 1.2.0
+    time ${Time.at((record["_SOURCE_REALTIME_TIMESTAMP"] || record["__REALTIME_TIMESTAMP"]).to_f / 1000000.0).to_datetime.rfc3339(6)}
```

Comment:
is this different from syslog_record_transform.conf line 32 because docker always fills one of these fields?